common/network-common/src/main/java/org/apache/spark/network/util/TransportConf.java

```diff
@@ -213,6 +213,11 @@ public boolean encryptionEnabled() {
     return conf.getBoolean("spark.network.crypto.enabled", false);
   }
 
+  /**
+   * Version number to be used by the AuthEngine key agreement protocol. Value values are 1 or 2.
```

Comment:
`Value values`?

Reply:
Also, shall we mentioned that the default is 1 intentionally?

Reply:
Good catch. Done.